src/lib/services/openapi-service.ts

```diff
@@ -38,12 +43,40 @@ export class OpenApiService {
     }
 
     validPath(op: ApiOperation): RequestHandler {
-        return this.api.validPath(op);
+        const { beta, enterpriseOnly, ...rest } = op;
+
+        const betaBadge = beta
+            ? `![Beta](${this.docsStaticsPath()}/Beta.svg)
+            
+            `
+            : '';
+        const enterpriseBadge = enterpriseOnly
+            ? `![Enterprise](${this.docsStaticsPath()}/Enterprise.svg) **Enterprise feature**
```

Comment:
Small nit, but I think this is a clearer alt text

```suggestion
            ? `![Unleash Enterprise](${this.docsStaticsPath()}/Enterprise.svg) **Enterprise feature**
```